hackclub.com.yaml

```
@@ -2128,6 +2128,11 @@ summer:
         preference: 10
       - exchange: aspmx4.googlemail.com.
         preference: 10
+stem-kfs:
+  - ttl: 1
+    type: CNAME
+    value: https://stem-kfs-website.vercel.app.
+    
```

There should be no protocol (https) in DNS records, maybe try something like this?

```suggestion
stem-kfs:
  - ttl: 1
    type: CNAME
    value: stem-kfs-website.vercel.app.
```

Additionally, a ttl of 1 seems to be really low -- 600 seconds (or 10 minutes) seems to be a better minimum.